main/constants/constants.ts

```diff
@@ -1,7 +1,4 @@
 export const DI_ROOT_INJECTOR_KEY = 'needle.morganstanley.com.root.injector';
-export const DI_GLOBAL_STATE_STORE = 'needle.morganstanley.com.state.store';
-export const GLOBAL_INSTANCE_MAP = 'needle.morganstanley.com.instance.map';
-export const GLOBAL_REGISTRATION_MAP = 'needle.morganstanley.com.registrations.map';
```

Comment:
Do we need to make this a major because of this (I would guess not as these can be considered internal)?
Are there concerns about compatibility between different versions of DI with these changes made?
I think we at least need a minor as new functionality is being added.

Reply:
This is all going on the 1.0.0 branch which will be a major.